tests/csm/rest/test_capacity_usage.py

```diff
@@ -1524,3 +1524,38 @@ def test_44347(self):
             assert result[0], f"{result[1]} for {failure_cnt} failures"
 
         self.deploy = True
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
```

Comment:
This test will not run in parallel. Can u remove parallel tag

Reply:
removed parallel tag